generator/annual-report/sig_report.tmpl

```diff
@@ -33,6 +33,7 @@
 -->
 
 4. KEP work in {{lastYear}} ({{$releases.LatestMinusTwo}}, {{$releases.LatestMinusOne}}, {{$releases.Latest}}):
+<!--
```

Comment:
Realised, we need to move [this section of code-block](https://github.com/kubernetes/community/blob/97526afbde0a6384e3465d13f8590271a8e16229/generator/annual-report/sig_report.tmpl#L17-L27) down as well and update it with a `TODO` instruction:

So, final update would look something like following (or check for the full updated file [here](https://raw.githubusercontent.com/Priyankasaggu11929/community/test-kep-auto-generation-with-comments/generator/annual-report/sig_report.tmpl)):

```go
{{$releases := getReleases -}}
{{$owningSIG := .Dir -}}
{{$stagedKeps := filterKEPs $owningSIG $releases -}}
{{$alphaKeps := index $stagedKeps "alpha" -}}
{{$betaKeps := index $stagedKeps "beta" -}}
{{$stableKeps := index $stagedKeps "stable" -}}
4. KEP work in {{lastYear}} ({{$releases.LatestMinusTwo}}, {{$releases.LatestMinusOne}}, {{$releases.Latest}}):
<!--
   TODO: Uncomment the following auto-generated list of KEPs, once reviewed & updated for correction.

   Note: This list is generated from the KEP metadata in kubernetes/enhancements repository.
      If you find any discrepancy in the generated list here, please check the KEP metadata.
      Please raise an issue in kubernetes/community, if the KEP metadata is correct but the generated list is incorrect.
-->

<!-- {{if ne (len $alphaKeps) 0}}
  - Alpha
  {{- range $alphaKeps}}
    - [{{.Number}} - {{.Title}}](https://github.com/kubernetes/enhancements/tree/master/keps/{{.OwningSIG}}/{{.Name}}) - {{.Milestone.Alpha -}}
  {{ end}}{{ end}}
{{if ne (len $betaKeps) 0}}
  - Beta
  {{- range $betaKeps}}
    - [{{.Number}} - {{.Title}}](https://github.com/kubernetes/enhancements/tree/master/keps/{{.OwningSIG}}/{{.Name}}) - {{.Milestone.Beta -}}
  {{ end}}{{ end}}
{{if ne (len $stableKeps) 0}}
  - Stable
  {{- range $stableKeps}}
    - [{{.Number}} - {{.Title}}](https://github.com/kubernetes/enhancements/tree/master/keps/{{.OwningSIG}}/{{.Name}}) - {{.Milestone.Stable -}}
  {{ end}}{{ end}} -->
```

Reply:
@Priyankasaggu11929 Done. Please verify

Reply:
@samsonkolge, here as well, please commit the exact code block I suggested above (comments & the code included)